src/content/community/releases/_index.en.md

```diff
@@ -3,6 +3,35 @@ date = 2020-08-24T11:35:16+02:00
 title = "Releases"
 weight = 40
 +++
+<!-- markdownlint-disable no-duplicate-heading/no-duplicate-header -->
```

Comment:
```suggestion
<!-- markdownlint-disable no-duplicate-header -->
```

I think any one of `MD024/no-duplicate-heading/no-duplicate-header` would work, but only one.

Reply:
Should we actually close it in the file? I imagine we want to use similar headers in the future as well...

Reply:
I was imagining opening it at the start of this file and closing it at the end, so any future heading would be included. I'm not sure what happens if we don't close it, maybe it's fine.

Reply:
Yeah, I think it's fine. And all tests are passing now as you can see, so the linter is happy.